patches/src/main/kotlin/app/revanced/patches/reddit/customclients/redditisfun/ads/Fingerprints.kt

```diff
@@ -0,0 +1,18 @@
+package app.revanced.patches.reddit.customclients.redditisfun.ads
+
+import app.revanced.patcher.fingerprint
+import com.android.tools.smali.dexlib2.AccessFlags
+
+// Should usually match: o5/i0.a()
+internal val userPremiumFingerprint = fingerprint {
```

Comment:
Obfuscated names will change for versions

```suggestion
internal val userPremiumFingerprint = fingerprint {
```